src/mumble/NetworkConfig.cpp

```diff
@@ -34,6 +35,8 @@ NetworkConfig::NetworkConfig(Settings &st) : ConfigWidget(st) {
 	qlePort->setAccessibleName(tr("Port"));
 	qleUsername->setAccessibleName(tr("Username"));
 	qlePassword->setAccessibleName(tr("Password"));
+
+	connect(qcbTcpMode, SIGNAL(stateChanged(int)), this, SIGNAL(on_qcbTcpMode_stateChanged(int)));
```

Comment:
The new syntax for connecting signals and slots (using function pointers instead of macros) should be preferred wherever possible.